plugins/apollo-explorer/README.md

```diff
@@ -38,17 +38,25 @@ yarn --cwd packages/app add @backstage/plugin-apollo-explorer
 Then, in `packages/app/src/App.tsx` add the plugin as a `Route`
 
 ```typescript
-<Route
-  path="/apollo-explorer"
-  element={
-    <ApolloExplorerPage
-      endpoints={[
-        { title: 'Github', graphRef: 'my-github-graph-ref@current' },
-        { title: 'Linear', graphRef: 'my-linear-graph-ref@current' },
-      ]}
+// In packages/app/src/App.tsx
+import { ApolloExplorerPage } from '@backstage/plugin-apollo-explorer';
+
+// You can add the tab to any number of pages, the service page is shown as an
+// example here
```

Comment:
```suggestion

```

These aren't tabs, but full pages. Opting to simply drop this as I think is a bit too core to Backstage to have in every plugin README